main/test/ca/uwaterloo/flix/library/TestReflect.flix

```diff
@@ -24,7 +24,7 @@ mod TestReflect {
 
     @Test
     def defaultBigDecimal01(): Unit \ Assert =
-        assertEq(expected = 0.0ff, (Reflect.default(): BigDecimal))
+        assertTrue(Object.isNull((Reflect.default(): BigDecimal)))
```

Comment:
Why in blazes do we have tests testing a property that we're not supposed to rely on?